FittedSheets/SheetContentViewController.swift

```diff
@@ -126,6 +139,45 @@ public class SheetContentViewController: UIViewController {
         self.contentWrapperView.layer.cornerRadius = self.treatPullBarAsClear ? 0 : self.cornerRadius
         self.childContainerView.layer.cornerRadius = self.treatPullBarAsClear ? self.cornerRadius : 0
     }
+
+    private var titleBarViewTopInset: CGFloat {
+        return (self.options.pullBarHeight + self.gripSize.height) / 2 + 2
```

Comment:
This math is a bit weird, but necessary because the design does not have the grip in the middle of the pullbar